### DISCUSSION TITLE
version from 04/11/23 - cannot change the ssh key

### DISCUSSION THREAD

**Discussion by lironsher** [iterate-ch/cyberduck#15341] [Q&A]
Hi
in version from 04/11/23
I'm creating a new profile for the ssh (scp) connection.

I'm selecting the key located in a folder on my drive (not .ssh)
and when I'm trying to connect it uses a different key from the .ssh folder. (the one that appears by default)

**Comment by dkocher** [comment]
Do you have configuration in `~/.ssh/config` for the host?
